UI/css/ledgersmb.css

```diff
@@ -73,11 +73,15 @@ table.warning {
     border: 0;
 }
 
-#company_info, #version_info, #login_info {
+#version_info {
    font-weight: bold;
    font-size: 9pt;
 }
 
+#company_info, #login_info {
```

Comment:
Don't quite understand the "kept distribution" part: there's a #version_info in expanding too